src/hotspot/cpu/riscv/vm_version_riscv.hpp

```diff
@@ -161,58 +199,146 @@ class VM_Version : public Abstract_VM_Version {
+  decl(ext_Zicsr        ,  Zicsr       ,  RV_NO_FLAG_BIT,  21,  true ,  NO_UPDATE_DEFAULT)                  \
+  decl(ext_Zicntr       ,  Zicntr      ,  RV_NO_FLAG_BIT,  22,  true ,  NO_UPDATE_DEFAULT)                  \
+  decl(ext_Zifencei     ,  Zifencei    ,  RV_NO_FLAG_BIT,  23,  true ,  NO_UPDATE_DEFAULT)                  \
+  decl(ext_Zic64b       ,  Zic64b      ,  RV_NO_FLAG_BIT,  24,  true ,  UPDATE_DEFAULT(UseZic64b))          \
+  decl(ext_Ztso         ,  Ztso        ,  RV_NO_FLAG_BIT,  25,  true ,  UPDATE_DEFAULT(UseZtso))            \
+  decl(ext_Zihintpause  ,  Zihintpause ,  RV_NO_FLAG_BIT,  26,  true ,  UPDATE_DEFAULT(UseZihintpause))     \
+  decl(ext_Zacas        ,  Zacas       ,  RV_NO_FLAG_BIT,  27,  true ,  UPDATE_DEFAULT(UseZacas))           \
+  decl(ext_Zvbb         ,  Zvbb        ,  RV_NO_FLAG_BIT,  28,  true ,  UPDATE_DEFAULT_DEP(UseZvbb, ext_V)) \
+  decl(ext_Zvbc         ,  Zvbc        ,  RV_NO_FLAG_BIT,  29,  true ,  UPDATE_DEFAULT_DEP(UseZvbc, ext_V)) \
+  decl(ext_Zvfh         ,  Zvfh        ,  RV_NO_FLAG_BIT,  30,  true ,  UPDATE_DEFAULT_DEP(UseZvfh, ext_V)) \
+  decl(ext_Zvkn         ,  Zvkn        ,  RV_NO_FLAG_BIT,  31,  true ,  UPDATE_DEFAULT_DEP(UseZvkn, ext_V)) \
+  decl(ext_Zicond       ,  Zicond      ,  RV_NO_FLAG_BIT,  32,  true ,  UPDATE_DEFAULT(UseZicond))          \
+
+  #define DECLARE_RV_EXT_FEATURE(NAME, PRETTY, LINUX_BIT, CPU_FEATURE_INDEX, FSTRING, FLAGF)                   \
+  struct NAME##RVExtFeatureValue : public RVExtFeatureValue {                                                  \
+    NAME##RVExtFeatureValue(const char* pretty, int linux_bit_num, uint32_t cpu_feature_index, bool fstring) : \
+      RVExtFeatureValue(pretty, linux_bit_num, cpu_feature_index, fstring) {}                                  \
+    FLAGF;                                                                                                     \
+  };                                                                                                           \
+  static NAME##RVExtFeatureValue NAME;                                                                         \
+
+  RV_EXT_FEATURE_FLAGS(DECLARE_RV_EXT_FEATURE)
+
+  // Non-extension features
+  //
+  #define RV_NON_EXT_FEATURE_FLAGS(decl)                                                                   \
+  decl(unaligned_access ,  Unaligned       ,  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
+  decl(mvendorid        ,  VendorId        ,  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
+  decl(marchid          ,  ArchId          ,  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
+  decl(mimpid           ,  ImpId           ,  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
+  decl(satp_mode        ,  SATP            ,  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
+  decl(zicboz_block_size, "ZicbozBlockSize",  RV_NO_FLAG_BIT,  false,  NO_UPDATE_DEFAULT)                  \
```

Comment:
Hmm, why is this `"ZicbozBlockSize"` different from friends? Do we need the double quotes?